metapackages/auto-configuration-propagators/README.md

```diff
@@ -0,0 +1,53 @@
+# OpenTelemetry Meta Package for Propagators Configuration
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-url]
+
+## About
+
+This module provides a way to get a propagator based on the [OTEL_PROPAGATORS environment variable](env-var-url).
```

Comment:
```suggestion
This module provides a way to get a propagator based on the [`OTEL_PROPAGATORS` environment variable](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#general-sdk-configuration).
```

... or pull that URL out to an id:

```md
... [blah blah][env-var-url]

... 
[env-var-url]: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#general-sdk-configuration
```

The same URL is also used below in the same file.

Reply:
updated